### DISCUSSION TITLE
Losing Background on Bartender4 Pet Bar

### DISCUSSION THREAD

**Discussion by lissaxjae** [SFX-WoW/Masque#194] [Issues] [answered]
Not sure if this is a B4 or Masque issue, but the background for the pet bar won't remain after any sort of reload. Using Caith and background enabled in global options. Bars 1 and 2 work fine.

Bartender4-4.8.2
Masque-8.0.1
Masque_Caith-8.0.0

**Comment by StormFX** [comment]
I'll look into it.

**Comment by StormFX** [comment, marked as answer]
This seems to be an issue with how the game (and subsequently, add-ons using the game's API) handles empty buttons for the pet bar. This will take some deeper diving to see if it's something I can fix with Masque or if it's something the add-ons themselves will have to handle.
